content/hi/docs/concepts/overview/components.md

```diff
@@ -0,0 +1,118 @@
+---
+reviewers:
+- lavalamp
+title: कुबेरनेट्स घटक
+content_type: concept
+description: >
+कुबेरनेट्स क्लस्टर में ऐसे घटक होते हैं जो नियंत्रण विमान का प्रतिनिधित्व करते हैं और मशीनों का एक सेट जिसे नोड्स कहा जाता है।
+weight: 20
+card: 
+  name: concepts
+  weight: 20
+--- 
+
+<!-- overview -->
+
+जब आप कुबेरनेट्स को तैनात करते हैं, तो आपको एक क्लस्टर मिलता है।
+{{< glossary_definition term_id="cluster" length="all" prepend="A Kubernetes cluster consists of">}}
```

Comment:
Should you create a Hindi glossary definition?

Reply:
https://kubernetes.io/docs/contribute/localization/#language-specific-style-guide-and-glossary

Reply:
@AmanSarraf  You can click the Resolved button for this comment (only you have the button). You've created the glossary.md file in this PR.